src/app/components/core/configuration-manager/configuration-group/tab-header-slider.ts

```diff
@@ -0,0 +1,140 @@
+      // Find how far this panel has been scrolled
+      var availableScrollLeft = this.groupNavigation.scrollLeft;
+      // If the space available is less than two lots of our desired distance, just move the whole amount
+      // otherwise, move by the amount in the settings
+      if (availableScrollLeft < SETTINGS.navBarTravelDistance * 2) {
+        this.groupNavContents.style.transform = "translateX(" + availableScrollLeft + "px)";
+      } else {
+        this.groupNavContents.style.transform = "translateX(" + SETTINGS.navBarTravelDistance + "px)";
+      }
+      // Show transition only in forward stroke
+      this.groupNavContents.style.transition = "transform 0.2s ease-in-out";
+      // Update settings
+      SETTINGS.navBarTravelDirection = "left";
+      SETTINGS.navBarTravelling = true;
+    }
+    // Now update the attribute in the DOM
+    this.groupNavigation.setAttribute("data-overflowing", this.determineOverflow(this.groupNavContents, this.groupNavigation));
+  }
+
+  scrollToRight() {
+    // If in the middle of a move return
+    if (SETTINGS.navBarTravelling === true) {
+      return;
+    }
+    // If we have content overflowing both sides or on the right
+    if (this.determineOverflow(this.groupNavContents, this.groupNavigation) === "right" || this.determineOverflow(this.groupNavContents, this.groupNavigation) === "both") {
+      // Get the right edge of the container and content
+      var navBarRightEdge = this.groupNavContents.getBoundingClientRect().right;
+      var navBarScrollerRightEdge = this.groupNavigation.getBoundingClientRect().right;
+      // Now we know how much space we have available to scroll
+      var availableScrollRight = Math.floor(navBarRightEdge - navBarScrollerRightEdge);
+      // If the space available is less than two lots of our desired distance, just move the whole amount
+      // otherwise, move by the amount in the settings
+      if (availableScrollRight < SETTINGS.navBarTravelDistance * 2) {
+        this.groupNavContents.style.transform = "translateX(-" + availableScrollRight + "px)";
+      } else {
+        this.groupNavContents.style.transform = "translateX(-" + SETTINGS.navBarTravelDistance + "px)";
+      }
+      // Show transition only in forward stroke
+      this.groupNavContents.style.transition = "transform 0.2s ease-in-out";
+      // Update settings
+      SETTINGS.navBarTravelDirection = "right";
+      SETTINGS.navBarTravelling = true;
+    }
+    // Now update the attribute in the DOM
+    this.groupNavigation.setAttribute("data-overflowing", this.determineOverflow(this.groupNavContents, this.groupNavigation));
+  }
+
+  setOverFlow() {
+    this.groupNavigation.setAttribute("data-overflowing", this.determineOverflow(this.groupNavContents, this.groupNavigation));
+  }
+
+
+  determineOverflow(content, container) {
```

Comment:
this.determineOverflow(this.groupNavContents, this.groupNavigation

should we keep 
```
determineOverflow() {
 content = this.this.groupNavContents;
 container =  this.groupNavigation;
  ....
```
This will allows us to make statement short and add more readability?